guides/workload-autoscaling/README.md

```diff
@@ -0,0 +1,300 @@
+# Autoscaling with Workload Variant Autoscaler (WVA)
+
+The [Workload Variant Autoscaler](https://github.com/llm-d-incubation/workload-variant-autoscaler) (WVA) provides dynamic autoscaling capabilities for llm-d inference deployments, automatically adjusting replica counts based on workload metrics and desired performance characteristics.
+
+## Overview
+
+WVA integrates with llm-d to:
+- Dynamically scale inference replicas based on workload metrics
+- Optimize resource utilization by adjusting to traffic patterns
+- Reduce tail latency through intelligent scaling decisions
+- Support multiple accelerator types
+
+> **Note**: WVA currently supports only the [Intelligent Inference Scheduling](../inference-scheduling/README.md) well-lit path. Other well-lit paths (such as Prefill/Decode Disaggregation or Wide Expert-Parallelism) are not currently supported.
+
+## Prerequisites
+
+Before installing WVA, ensure you have:
+
+1. **Gateway control plane**: Configure and deploy your [Gateway control plane](../prereq/gateway-provider/README.md) (Istio) before installation.
+
+2. **Prometheus monitoring stack**: WVA requires Prometheus to be accessible for metric collection. **WVA requires HTTPS connections to Prometheus** - HTTP-only Prometheus will cause WVA to fail. The monitoring setup depends on your platform:
+   - **OpenShift**: User Workload Monitoring should be enabled (see [OpenShift monitoring docs](https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/monitoring/configuring-user-workload-monitoring))
+   - **GKE**: An in-cluster Prometheus instance is required (GMP does not expose HTTP API). See [GKE configuration](#gke) below for setup instructions.
+   - **Kind/Minikube**: Prometheus must be configured with TLS/HTTPS. Default HTTP-only Prometheus will not work with WVA. See [Kind/Minikube configuration](#other-kubernetes-platforms-kind-minikube-etc) below.
```

Comment:
This is not very clear as it doesn't say how to install the monitoring stack in Kind/Minikube. I suggest to merge it with the bullet text below.

Reply:
updated